### PR TITLE
Template Animations: Added float-on animation (#618)

### LINKED ISSUE #618: Basic Template Animations
## Feature Description

As an author I want to be able to use animations from a template

**[UX](https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Google-Stories---Templates-(Copy)?node-id=1%3A5415)**

**Awaiting on feature brief for Animations to be written**

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

1. As an author I am able to save a Bounce animation as a template animation
2. As an author I am able use a previously saved Bounce template animation
1. As an author I am able to save a Text Border animation as a template animation
2. As an author I am able use a previously saved Text Border template animation
1. As an author I am able to save a Text March animation as a template animation
2. As an author I am able use a previously saved Text March template animation
1. As an author I am able to save a Text Border March animation as a template animation
2. As an author I am able use a previously saved Text Border March template animation
1. As an author I am able to save a Text Reveal animation as a template animation
2. As an author I am able use a previously saved Text Reveal template animation
1. As an author I am able to save a Spin animation as a template animation
2. As an author I am able use a previously saved Spin template animation

### PR DESCRIPTION
**Pull request by mariano-formidable**
This PR adds a very simple `floatOn` animation to our library of template animations.

Bottom to top:
float-on-default

Top to bottom:
float-on-top-to-bottom

Left to right:
float-on-left-to-right

Right to left:
float-on-right-to-left

### CODE REVIEW

**Review by BrittanyIRL — APPROVED** [review]
Awesome! Super straight forward and easy to follow. Matches existing patterns set up.

**Review by carloskelly13 — APPROVED** [review]

**Comment by mariano-formidable** [comment]
@pbakaus Can you merge this PR when you get a chance?  I don't seem to have merge permissions 😱

**Comment by BrittanyIRL** [comment]
> @pbakaus Can you merge this PR when you get a chance? I don't seem to have merge permissions 😱

I have merge permissions for this one - merged! Though, @pbakaus while we have you - is there some setting that can change so that this is easier on future prs?

**Comment by pbakaus** [comment]
Mhh weird, you two have the same permissions? Maybe @swissspidy knows more..

**Pull request merged**
